app/controllers/reviews_controller.rb

```diff
@@ -1,27 +1,49 @@
 class ReviewsController < ApplicationController
+  before_action :authenticate_user!, except: [:index]
```

Comment:
You don't have an index action for reviews - so why are you specifying that you shouldn't authenticate user for that action?